test/k8s-integration/driver-config.go

```diff
@@ -38,10 +40,41 @@ func generateDriverConfigFile(pkgDir, storageClassFile string) (string, error) {
 	defer w.Flush()
 
 	// Fill in template parameters
+	caps := []string{
+		"persistence",
+		"block",
+		"fsGroup",
+		"exec",
+		"multipods",
+		"topology",
+	}
+
+	switch deploymentStrat {
+	case "gke":
+	case "gce":
+		// TODO: OSS K8S supports volume expansion for CSI by default in 1.16+;
```

Comment:
Should we consider K8s version specific capability setting? ie if we're running on a 1.16+ cluster, then enable resizing tests.

Reply:
Do we need it today? Should we try adding it later?

Reply:
We can do it later but it means that we will be lacking test coverage of these new features we're releasing until GKE can catch up.

Reply:
Is there a subset of coverage we can get on GKE before k8s 1.16 is available?